src/gemdat/plots/plotly/_displacements.py

```diff
@@ -66,6 +66,62 @@ def displacement_per_element(*, trajectory: Trajectory) -> go.Figure:
     return fig
 
 
+def msd_per_element(*, trajectory: Trajectory) -> go.Figure:
+    """Plot mean squared displacement per element.
+
+    Parameters
+    ----------
+    trajectory : Trajectory
+        Input trajectory
+
+    Returns
+    -------
+    fig : matplotlib.figure.Figure
+        Output figure
+    """
+
+    fig = go.Figure()
+
+    grouped = defaultdict(list)
+
+    species = trajectory.species
+
+    for sp, distances in zip(species,
+                             trajectory.distances_from_base_position()):
+        grouped[sp.symbol].append(distances**2)
+
+    for symbol, sq_distances in grouped.items():
+        mean_sq_disp = np.mean(sq_distances, axis=0)
+        std_disp = np.std(sq_distances, axis=0)
```

Comment:
Calculating the std in this way can result in a negative MSD, which is (unless the displacements can be imaginary) not quite correct.

![newplot](https://github.com/GEMDAT-repos/GEMDAT/assets/1472570/8dafa58e-677a-4488-b5c2-721622266b84)

I think it might not make much sense to include the std here, but I leave this up to you